extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -585,6 +586,20 @@ export class VariantAnalysisManager
     await cancelVariantAnalysis(credentials, variantAnalysis);
   }
 
+  public async openVariantAnalysisLogs(variantAnalysisId: number) {
```

Comment:
We could add an integration test for this function, but it doesn't have to be part of this PR.

Reply:
Good point. I'll look at doing that in a followup